### DISCUSSION TITLE
v-model for Splitter size

### DISCUSSION THREAD

**Discussion by xaviergmail** [primefaces/community#1698] [PrimeVue]
I'd like to request a v-model binding to be able to control a Splitter's sizes. This v-model would preferably be on the `<Splitter>` itself to control _all_ `sizes`, but I would also be happy with controlling it on each <SplitterPanel> as well.

My use case, is I have a TabView with 3 tabs, each one has a splitter. I'd like to be able to synchronize the splitters for each tab.

**Comment by StereoApp** [comment]
I agree that size would be very usable if it were reactive, until the implementation, you can try the following solution, this is works for me.
```JavaScript
<script setup lang="ts">
    const size = ref<number>(10)
</script>
<template>
  <Splitter h="full" w="full" @resize="({ sizes }) => { size = sizes[1] }">
      <SplitterPanel :size="100 - size" />
      <SplitterPanel :size="size" :pt:root:style="{ flexBasis: `calc(${size}% - 4px)` }" />
  </Splitter>
</template>
```
I'm not sure if this is a good practice, just to share ideas and hopefully help you.
